lib/playwright/locator_assertions_impl.rb

```diff
@@ -0,0 +1,413 @@
+module Playwright
+  # ref: https://github.com/microsoft/playwright-python/blob/main/playwright/_impl/_assertions.py
+  define_api_implementation :LocatorAssertionsImpl do
+    def self.define_negation(method_name)
+      define_method(:"not_#{method_name}") do |*args, **kwargs|
+        self.not.send(method_name, *args, **kwargs)
```

Comment:
For Ruby < 2.7 compatibility, we would have to write like this for avoiding `{}` passed unexpectedly.

```
if kwargs.empty?
  self.not.public_send(method_name, *args)
else
  self.not.public_send(method_name, *args, **kwargs)
end
```